src/fsharp/lib.fs

```diff
@@ -226,9 +226,8 @@ module ListSet =
         | (h::t) -> if contains f h l1 then h::intersect f l1 t else intersect f l1 t
         | [] -> []
 
-    (* NOTE: quadratic! *)
```

Comment:
Why was the comment removed? It is still quadratic.

Reply:
My mistake, sorry. While it is faster, it is clearly still quadratic. This has already been merged, do you want a PR to bring the comment back?